### PR TITLE
Clean up the repo

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Bumps to `tomte@v0.2.12` and cleans up the repo.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -473,6 +473,9 @@ ignore_errors=True
 [mypy-packages.fetchai.*]
 ignore_errors=True
 
+[mypy-packages.valory.*]
+ignore_errors=True
```
Comment (Adamantios) [inline tox.ini:477]: The repo had the same issue as in https://github.com/valory-xyz/governatooorr/issues/32.

File: setup.cfg [hunk 2]
```diff
@@ -1,101 +0,0 @@
...
-[mypy]
-python_version = 3.8
-strict_optional = True
-exclude=(.*_pb2|.*custom_types)
-
-# Before adding a module here, make sure it does not support type hints
-# Per-module options for aea dir:
-
-[mypy-packages.valory.*]
-ignore_errors=True
```
Comment (Adamantios) [inline setup.cfg:50]: Reason for #54.

**Review by 0xArdi — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (dvilelaf) [reply tox.ini:477]: Don't the other repos have this issue? Aren't we using the same approach everywhere?

Comment (Adamantios) [reply tox.ini:477]: I only found this on IEKit and governatooorr.

**Review by dvilelaf — APPROVED** [review]

**Pull request merged**